### ISSUE TITLE
Interface locked up while uploading

### ISSUE THREAD

**Issue by GnaXi** [Ylianst/MeshCentral#639]
Currently when using the Files feature and uploading a file to a system, the entire meshcentral interface is locked.
I would love to be able to navigate around meshcentral while a file is being uploaded, instead of being forced to open a separate browser window! :)

**Comment by Ylianst** [comment]
Agreed. This is a good request. On my test server, all transfers are on LAN and so, always very quick. I can see how this could be a problem.

**Comment by MailYouLater** [comment]
While technically what I'm requesting should be a separate feature request, I thought I'd mention it here because the implementation of it may be related to how you implement this.
I've noticed that the dialogs that pop up are stuck in the center of the screen area, and aren't movable.  This has been an annoyance for me when I wanted to reference something behind it after it opened, e.g. while typing something into the "Remote Keyboard Entry" dialog.  I don't know how difficult it would be to implement, but it would be really nice if you could grab the title bar of one of these dialogs and drag it around.
